packages/next/src/server-build.ts

```diff
@@ -128,11 +133,21 @@ export async function serverBuild({
   prerenderManifest: NextPrerenderedRoutes;
   requiredServerFilesManifest: NextRequiredServerFilesManifest;
 }): Promise<BuildResult> {
+  Object.assign(lambdaPages, lambdaAppPaths);
```

Comment:
Is it safe to mutate the `lambdaPages` here? What if the calling function uses it after calling this function?

Reply:
Should be safe as the builder returns the result of the server build directly. Updated to create a separate object though.